.idea/misc.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
Does this actually still happen to you even with recent changes in develop? I thought I fixed this, or at least it doesn't seem to be happening to me anymore. Can you try reverting this change, clean & build the project and see if it comes back? If it does, let's check it in. If not, let's avoid it since I hope it's unnecessary.

Reply:
okay